react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-import Game from './screens/Game';
+import MatchesService from '../services/MatchesService';
 
+import Game from './screens/Game';
+import Matches from './screens/Matches';
 import '../scss/application.scss';
+import styles from './styles.module.scss';
+
+class App extends Component {
+  state = {
+    data: []
```

Comment:
This `data` array is declared here but only used by `Matches` component, so it should be defined in that scope.